### DISCUSSION TITLE
Playtime 2: Option for stop button behavior on columns to end recording without looping

### DISCUSSION THREAD

**Discussion by sitruss** [helgoboss/helgobox#1385] [Ideas]
Something I find very useful from Ableton but can't seem to work out in Playtime: 

When I'm currently recording to a cell in session view and press the column's stop button, it finishes recording in that cell _without playing back_ so that I can "store" the stopped loop to bring back later. This is super useful for ideas that don't repeat the first time they appear in the music but will be played as a regular loop later in the composition. I also think it's a bit more intuitive: the stop button stops rather than plays, even while recording.

**Comment by helgoboss** [comment]
So you would like the recorded clip to not play when pressing the column stop button or the matrix stop button but to play when pressing the slot record button or smart-record button?

**Reply by sitruss** [reply]
Yes! I think that workflow makes sense. I've always experienced "red record button toggles play, stop button stops" in most tools I can remember, though things like Ableton's looper device let you set the red recording button behavior to either play or go straight into overdubbing. If audio overdubbing is added in the future the record button would need revisiting, but for now making the column/matrix stop buttons work as described above would be awesome.
